packages/sui-ssr/server/dynamicRendering/index.js

```diff
@@ -2,17 +2,20 @@ import path from 'path'
 import fs from 'fs'
 import seoBotDetect from './seoBotDetect'
 
-const {DYNAMIC_RENDERING = false} = process.env
 const INDEX_HTML_PATH = path.join(process.cwd(), 'public', 'index.html')
 
-export default function dinamycRendering(fallback) {
+export default function dynamicRendering(fallback, dynamicsURLS = []) {
   return function middleware(req, resp, next) {
-    if (!DYNAMIC_RENDERING) {
+    if (!dynamicsURLS.length || seoBotDetect(req)) {
```

Comment:
Awesome. I like this middleware code, it's very neat! Congrats! 👏

Reply:
JAJAJAJA. I HATE YOU!!!